tests/devices/test_default_qutrit.py

```diff
@@ -72,6 +72,152 @@ class TestApply:
     """
 
     # TODO: Add tests for non-parametric ops after they're implemented
+    test_data_no_parameters = [
+        (qml.TShift, [1, 0, 0], np.array([0, 1, 0]), None),
+        (
+            qml.TShift,
+            [1 / math.sqrt(2), 1 / math.sqrt(2), 0],
+            np.array([0, 1 / math.sqrt(2), 1 / math.sqrt(2)]),
+            None,
+        ),
+        (qml.TClock, [1, 0, 0], np.array([1, 0, 0]), None),
+        (qml.TClock, [0, 1, 0], np.array([0, OMEGA, 0]), None),
+    ]
+
+    test_data_no_parameters_inverses = [
+        (qml.TShift, [0, 1, 0], np.array([1, 0, 0]), None),
+        (
+            qml.TShift,
+            [0, 1 / math.sqrt(2), 1 / math.sqrt(2)],
+            np.array([1 / math.sqrt(2), 1 / math.sqrt(2), 0]),
+            None,
+        ),
+        (qml.TClock, [1, 0, 0], np.array([1, 0, 0]), None),
+        (qml.TClock, [0, OMEGA, 0], np.array([0, 1, 0]), None),
+    ]
+
+    @pytest.mark.parametrize("operation, input, expected_output, subspace", test_data_no_parameters)
+    def test_apply_operation_single_wire_no_parameters(
+        self, qutrit_device_1_wire, tol, operation, input, expected_output, subspace
+    ):
+        """Tests that applying an operation yields the expected output state for single wire
+        operations that have no parameters."""
+
+        qutrit_device_1_wire._state = np.array(input, dtype=qutrit_device_1_wire.C_DTYPE)
+        qutrit_device_1_wire.apply(
+            [operation(wires=[0]) if subspace is None else operation(wires=[0], subspace=subspace)]
+        )
+
+        assert np.allclose(qutrit_device_1_wire._state, np.array(expected_output), atol=tol, rtol=0)
+        assert qutrit_device_1_wire._state.dtype == qutrit_device_1_wire.C_DTYPE
+
+    @pytest.mark.parametrize(
+        "operation,input,expected_output, subspace", test_data_no_parameters_inverses
+    )
+    def test_apply_operation_single_wire_no_parameters_inverse(
+        self, qutrit_device_1_wire, tol, operation, input, expected_output, subspace
+    ):
+        """Tests that applying an operation yields the expected output state for single wire
+        operations that have no parameters."""
+
+        qutrit_device_1_wire._state = np.array(input, dtype=qutrit_device_1_wire.C_DTYPE)
+        qutrit_device_1_wire.apply(
+            [
+                operation(wires=[0]).inv()
+                if subspace is None
+                else operation(wires=[0], subspace=subspace).inv()
+            ]
+        )
+
+        assert np.allclose(qutrit_device_1_wire._state, np.array(expected_output), atol=tol, rtol=0)
+        assert qutrit_device_1_wire._state.dtype == qutrit_device_1_wire.C_DTYPE
+
+    test_data_two_wires_no_parameters = [
```

Comment:
For all three of these, I recommend to add an additional test case with complex values and 1-2 more terms in the superposition just to ensure that things aren't coming out correct just due to some symmetry in the input.

Reply:
Resolving this comment as it looks like these have been added. Nice work :star: